pkgs/development/node-packages/default.nix

```diff
@@ -297,6 +297,17 @@ let
       meta.mainProgram = "postcss";
     };
 
+    prisma = super.prisma.override {
+      nativeBuildInputs = [ pkgs.makeWrapper ];
+      postInstall = ''
+        wrapProgram "$out/bin/prisma" \
+          --prefix PRISMA_MIGRATION_ENGINE_BINARY : "${pkgs.prisma-engines}/bin/migration-engine" \
+          --prefix PRISMA_QUERY_ENGINE_BINARY : "${pkgs.prisma-engines}/bin/query-engine" \
+          --prefix PRISMA_INTROSPECTION_ENGINE_BINARY : "${pkgs.prisma-engines}/bin/introspection-engine" \
+          --prefix PRISMA_FMT_BINARY : "${pkgs.prisma-engines}/bin/prisma-fmt"
```

Comment:
The next version (hopefully) will set the Node API to be the default interface for the Query Engine. You want to add that variable too to the wrapping:

```
           --prefix PRISMA_QUERY_ENGINE_LIBRARY= : "${pkgs.prisma-engines}/lib/libquery_engine.so.node"
```

This is a bit tricky, due to it being named `libquery_engine.dylib.node` on macOS and `query_engine.dll.node` on Windows. But without having it the Prisma Client will not start if (for now) the `nApi` preview feature is turned on, and from the next version forward it will just not work at all.

Reply:
When you build the engines, the Node API library will be in `target/release/libquery_engine.so` on Linux. You have to move it from there AND rename it to have `.node` file extension for NodeJS to be able to load it.

Reply:
I'm also thinking, the VSCode extension uses the `prisma-fmt` (for now). We're quite soon just getting rid of it and compiling a better version as WASM, but for now if you use the VSCode extension, it should have a working `prisma-fmt` somewhere available.

Reply:
I don't use VSCode, and I have no ideas how the extensions are packaged in `nixpkgs`.
I will leave it aside for now, it can go in a separate PR.